src/index.ts

```diff
@@ -1,11 +1,14 @@
 export * from './use-swr'
 import { default as useSWR } from './use-swr'
 export { useSWRPages } from './use-swr-pages'
+export { cache } from './config'
 export {
   ConfigInterface,
   revalidateType,
   RevalidateOptionInterface,
   keyInterface,
-  responseInterface
+  responseInterface,
+  CacheInterface
 } from './types'
+export { default as Cache } from './cache'
```

Comment:
Just thought more about this. It's still unclear to me why do we need to export the class `Cache`. Seems we can have support for multilayer cache, async cache and namespaced cache on top of the singleton `cache`.

Reply:
I did it because the original implementation let you create your own instance, right now it doesn't make any sense, I remove it.

Reply:
Done in https://github.com/zeit/swr/pull/231/commits/92d6ec748bbdab088c462a54455ba379437c1a29